### ISSUE TITLE
#555 regression in node 11?

### ISSUE THREAD

**Issue by simonbuchan** [nexe/nexe#596]
**Is this a BUG or a FEATURE REQUEST?:** 
Bug

**What happened:**
Seems to be a regression as in #555, `test.exe --test` gives `test.exe: bad option: --test`, when built with `nexe -t 11.10.0 --build`

**What you expected to happen:**
Passed to `process.argv`.

**How to reproduce it (as minimally and precisely as possible):**
Same as before:
```
>echo "console.log(process.argv)" | node_modules\.bin\nexe -t 10.12.0 -b -
o test.exe && test.exe --test
i nexe 3.0.0-beta.15
√ Source already downloaded
√ Compiling result
√ Entry: '[stdin]' written to: test.exe
√ Finished in 0.138s
[ '<snip>\\test.exe',
  '<snip>\\__nexe_stdin.js',
  '--test' ]
```

vs.:

```
>echo "console.log(process.argv)" | node_modules\.bin\nexe -t 11.10.0 -b -
o test.exe && test.exe --test
i nexe 3.0.0-beta.15
√ Source already downloaded
√ Compiling result
√ Entry: '[stdin]' written to: test.exe
√ Finished in 0.303s
test.exe: bad option: --test
```

**Anything else we need to know?:**
Seem to be able to use `--` to separate the node and script parameters:

```
>test.exe --inspect -- --test
Debugger listening on ws://127.0.0.1:9229/458a44e0-6a9e-4cd4-a1e9-42c2bad94ab4
For help, see: https://nodejs.org/en/docs/inspector
[ 'C:\\code\\skilitics\\interact\\new-player\\clients\\vrservice\\test.exe',
  'C:\\code\\skilitics\\interact\\new-player\\clients\\vrservice\\__nexe_stdin.js',
  '--test' ]
```

This seems like a handy enough option to support deliberately?

**Environment**

- Platform(OS/Version): Windows 10
- Host Node Version: 11.10.0
- Target Node Version: 11.10.0
- Nexe version: 3.0.0-beta.15
- Python Version: 2.7.15

**Comment by calebboyd** [comment]
Present in node > 11.6.0

**Comment by calebboyd** [comment]
closed via 37192e4ca852f3c968e5b55bb3dd55b229df5689 available with `next@next`
